src/main/scala/utilities/Attempto.scala

```diff
@@ -0,0 +1,166 @@
+package utilities
+import util.control.Breaks._
+
+/**
+ * A helper file that provides natural language methods for LISA.
+ * Usage:
+ * <pre>
+ * import utilities.Attempto.*
+ * </pre>
+ */
+object Attempto {
+
+    /**
+     * Helper for the aceToTptp function.
+     *
+     * @param URL String
+     * @return the corresponding content (String)
+     */
+    def get(url: String) = scala.io.Source.fromURL(url).mkString
+
+    /**
+     * Send a request to the Attempto Webservice and get the TPTP formula.
+     *
+     * @param ACE (Attempto Controlled English) String
+     * @return the TPTP formula (String) corresponding to the ACE input
+     */
+    def aceToTptp(text: String) : String = {
+     var query = text.replaceAll("\\s", "+")
+     return get(s"http://attempto.ifi.uzh.ch/ws/ape/apews.perl?text=$query&ctptp=on&solo=tptp")
+    }
+
+    /**
+     * Helper for the tptpToAce function.
+     * Decides if the appended string should be capitalized.
+     *
+     * @param String to be appended and the original String
+     * @return String to be appended with proper first letter case
+     */
+    def appendFormatStr(appendStr : String, originalStr : String) : String = {
+      var penultimateChar = 'c'
+      var resultStr = ""
+      if (originalStr.length > 2) {
+        penultimateChar = originalStr(originalStr.length-2)
+      }
+      if (penultimateChar == ':') {
+        resultStr = originalStr + appendStr.capitalize + ' '
+      } else {
+        resultStr = originalStr + appendStr + ' '
+      }
+      return resultStr
+    }
+
+    /**
+     * Helper for the tptpToAce function.
+     * Extracts the String from the matched Option[String].
+     *
+     * @param Regular expression for pattern matching, the matched String
+     * @return Tuple with the extracted String (if any) and a bool for the error code
+     */
+     def extractString(regex : scala.util.matching.Regex, matchedStr : String) : (String, Boolean) = {
+       var resultStr = ""
+       var errorCode = false
+       try {
+         resultStr = regex.findFirstIn(matchedStr).get
+       } catch {
+         case error:NoSuchElementException => errorCode = true
+       }
+       return (resultStr, errorCode)
+     }
```

Comment:
well, if you know for sure that your option is nonempty, you can use myOption.get. Otherwise, you can do something like
myOption match {
    case Some(value) => value
    case None => ""
}

Or even better: myOption.getOrElse("")